### PR TITLE
Update dev logs to include request timings

### PR DESCRIPTION
**Pull request by ijjk**
This updates our dev logs to include request timings and also includes fetch timings in app directory to including info on whether the cache was leveraged or not allowing easier debugging of contributing factors to render times. In the future we can expand to include middleware timings separate as well to single that out along with timings for `getStaticProps` potentially as well. 

<details>

<summary>Screenshot</summary>

CleanShot 2023-05-04 at 00 13 19@2x

</details>

<details>

<summary>Screenshot</summary>

CleanShot 2023-05-04 at 00 13 32@2x

</details>

<details>

<summary>Screenshot</summary>

CleanShot 2023-05-04 at 00 13 47@2x

</details>

x-ref: [slack thread](https://vercel.slack.com/archives/C042LHPJ1NX/p1681418590738249)

### CODE REVIEW

**Comment by ijjk** [comment]
## Failing test suites 
Commit: 3f200f13d10123c7f89be3ab0b352f8e14900b39

`pnpm testheadless test/development/api-route-errors/index.test.ts` 
- api-route-errors cli output > error
- api-route-errors cli output > uncaught exception
- api-route-errors cli output > unhandled rejection

<details>
<summary>Expand output</summary>

  ● api-route-errors cli output › error

    expect(received).toContain(expected) // indexOf

    Expected substring: "error - pages/api/error.js (2:8) @ error"
    Received string:    "- info Using locally built binary of @next/swc
    - event compiled client and server successfully in 2.9s (151 modules)
    - wait compiling /api/error (client and server)...
    - event compiled successfully in 129 ms (34 modules)
    - error pages/api/error.js (2:8) @ error
    - error Error

      at error (webpack-internal:///(api)../../pages/api/error.js:6:11)
      at ../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/api-utils/node.js:462:16
      at ../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/lib/trace/tracer.js:113:36
      at NoopContextManager.with (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:7057)
      at ContextAPI.with (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:516)
      at NoopTracer.startActiveSpan (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:18086)
      at ProxyTracer.startActiveSpan (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:18847)
      at ../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/lib/trace/tracer.js:102:107
      at NoopContextManager.with (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:7057)
      at ContextAPI.with (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:516)
      at NextTracerImpl.trace (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/lib/trace/tracer.js:102:32)
      at apiResolver (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/api-utils/node.js:460:63)
      at async DevServer.runApi (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/next-server.js:658:9)
      at async Object.fn (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/next-server.js:1124:35)
      at async Router.execute (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/router.js:315:32)
      at async DevServer.runImpl (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/base-server.js:601:29)
      at async DevServer.run (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/dev/next-dev-server.js:922:20)
      at async DevServer.handleRequestImpl (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/base-server.js:533:20) {
        digest: undefined
      }
        1 | export default function error(req, res) {
      > 2 |   throw new Error()
          |        ^
        3 | }
        4 |·
      - wait compiling /_error (client and server)...
      - event compiled client and server successfully in 1344 ms (160 modules)
      "
      at Object.<anonymous> (development/api-route-errors/index.test.ts:27:20)

  ● api-route-errors cli output › uncaught exception

    expect(received).toContain(expected) // indexOf

    Expected substring: "error - pages/api/uncaught-exception.js (3:10) @ Timeout"
    Received string:    "- GET /api/error 500 in 1655ms
    - wait compiling /api/uncaught-exception (client and server)...
    - event compiled successfully in 39 ms (35 modules)
    - GET /api/uncaught-exception 200 in 11ms
    - error pages/api/uncaught-exception.js (3:10) @ Timeout.eval [as _onTimeout]
    - error uncaughtException: Error: uncaught exception

      at Timeout.eval [as _onTimeout] (webpack-internal:///(api)../../pages/api/uncaught-exception.js:7:15)
      at process.processTimers (../node:internal/timers:512:7) {
        digest: undefined
      }
        1 | export default function uncaughtException(req, res) {
        2 |   setTimeout(() => {
      > 3 |     throw new Error('uncaught exception')
          |          ^
        4 |   }, 0)
        5 |   res.send('hello')
        6 | }
      "
      at Object.<anonymous> (development/api-route-errors/index.test.ts:41:20)

  ● api-route-errors cli output › unhandled rejection

    expect(received).toContain(expected) // indexOf

    Expected substring: "error - pages/api/unhandled-rejection.js (2:17) @ unhandledRejection"
    Received string:    "- wait compiling /api/unhandled-rejection (client and server)...
    - event compiled successfully in 21 ms (36 modules)
    - GET /api/unhandled-rejection 200 in 5ms
    - error pages/api/unhandled-rejection.js (2:17) @ unhandledRejection
    - error unhandledRejection: Error: unhandled rejection

      at unhandledRejection (webpack-internal:///(api)../../pages/api/unhandled-rejection.js:6:20)
      at ../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/api-utils/node.js:462:16
      at ../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/lib/trace/tracer.js:113:36
      at NoopContextManager.with (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:7057)
      at ContextAPI.with (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:516)
      at NoopTracer.startActiveSpan (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:18086)
      at ProxyTracer.startActiveSpan (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:18847)
      at ../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/lib/trace/tracer.js:102:107
      at NoopContextManager.with (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:7057)
      at ContextAPI.with (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:516)
      at NextTracerImpl.trace (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/lib/trace/tracer.js:102:32)
      at apiResolver (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/api-utils/node.js:460:63)
      at async DevServer.runApi (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/next-server.js:658:9)
      at async Object.fn (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/next-server.js:1124:35)
      at async Router.execute (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/router.js:315:32)
      at async DevServer.runImpl (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/base-server.js:601:29)
      at async DevServer.run (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/dev/next-dev-server.js:922:20)
      at async DevServer.handleRequestImpl (../../tmp/next-install-4c2c3100082f8cbb92fbfe362dbd036448f88b185d02166c6b3c2f6e76700563/node_modules/.pnpm/file+..+next-repo-43f922bcf3cf04df9b40163fac5b3c98_giqb3ocvb7qsxpbwpflyul3qja/node_modules/next/dist/server/base-server.js:533:20) {
        digest: undefined
      }
        1 | export default function unhandledRejection(req, res) {
      > 2 |   Promise.reject(new Error('unhandled rejection'))
          |                 ^
        3 |   res.send('hello')
        4 | }
        5 |·
      "
      at Object.<anonymous> (development/api-route-errors/index.test.ts:59:20)

</details>


Read more about building and testing Next.js in [contributing.md](https://github.com/vercel/next.js/blob/canary/contributing.md).

**Comment by ijjk** [comment]
## Failing test suites 
Commit: 6e87c54dccad6a73da2d5ce8b12b6e9e60ceec87

`pnpm testheadless test/integration/appdir-missing-config/test/index.test.ts` 
- Error when app dir is present without experimental.appDir > next build > should print warning when missing config with app and pages
- Error when app dir is present without experimental.appDir > next dev > should print warning when missing config with app and pages

<details>
<summary>Expand output</summary>

  ● Error when app dir is present without experimental.appDir › next dev › should print warning when missing config with app and pages

    expect(received).toMatch(expected)

    Expected substring: "warn  - The `app` directory is experimental. To enable, add `appDir: true` to your `next.config.js` configuration under `experimental`. See https://nextjs.org/docs/messages/experimental-app-dir-config"
    Received string:    "- ready started server on 0.0.0.0:39551, url: http://localhost:39551
    - warn The `app` directory is experimental. To enable, add `appDir: true` to your `next.config.js` configuration under `experimental`. See https://nextjs.org/docs/messages/experimental-app-dir-config
    - warn The `app` directory is experimental. To enable, add `appDir: true` to your `next.config.js` configuration under `experimental`. See https://nextjs.org/docs/messages/experimental-app-dir-config
    - info Using locally built binary of @next/swc
    - event compiled client and server successfully in 2.2s (150 modules)
    "

      26 |     await fs.outputFile(pagesIndex, 'module.exports = "index"')
      27 |     const output = await justPutIt()
    > 28 |     expect(output).toMatch(`warn  - ${msg}`)
         |                    ^
      29 |   })
      30 |   it('should not print when config found with app', async () => {
      31 |     await fs.writeFile(

      at Object.<anonymous> (integration/appdir-missing-config/test/index.test.ts:28:20)

  ● Error when app dir is present without experimental.appDir › next build › should print warning when missing config with app and pages

    expect(received).toMatch(expected)

    Expected substring: "warn  - The `app` directory is experimental. To enable, add `appDir: true` to your `next.config.js` configuration under `experimental`. See https://nextjs.org/docs/messages/experimental-app-dir-config"
    Received string:    "- info Linting and checking validity of types...
    - info Creating an optimized production build...
    - info Using locally built binary of @next/swc
    - info Compiled successfully
    - info Collecting page data...
    - warn The `app` directory is experimental. To enable, add `appDir: true` to your `next.config.js` configuration under `experimental`. See https://nextjs.org/docs/messages/experimental-app-dir-config
    - warn The `app` directory is experimental. To enable, add `appDir: true` to your `next.config.js` configuration under `experimental`. See https://nextjs.org/docs/messages/experimental-app-dir-config
    - warn The Next.js plugin was not detected in your ESLint configuration. See https://nextjs.org/docs/basic-features/eslint#migrating-existing-config·
    > Build optimization failed: found page without a React Component as default export in·
    pages/·
    See https://nextjs.org/docs/messages/page-without-valid-component for more info.·
    "

      26 |     await fs.outputFile(pagesIndex, 'module.exports = "index"')
      27 |     const output = await justPutIt()
    > 28 |     expect(output).toMatch(`warn  - ${msg}`)
         |                    ^
      29 |   })
      30 |   it('should not print when config found with app', async () => {
      31 |     await fs.writeFile(

      at Object.<anonymous> (integration/appdir-missing-config/test/index.test.ts:28:20)

</details>


Read more about building and testing Next.js in [contributing.md](https://github.com/vercel/next.js/blob/canary/contributing.md).

File: packages/next/src/build/output/log.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  wait: chalk.cyan('wait') + '  -',
-  error: chalk.red('error') + ' -',
-  warn: chalk.yellow('warn') + '  -',
-  ready: chalk.green('ready') + ' -',
-  info: chalk.cyan('info') + '  -',
-  event: chalk.magenta('event') + ' -',
-  trace: chalk.magenta('trace') + ' -',
+  wait: '- ' + chalk.cyan('wait'),
+  error: '- ' + chalk.red('error'),
+  warn: '- ' + chalk.yellow('warn'),
```
[the remark this answers is not in the document]
Comment (ijjk) [reply packages/next/src/build/output/log.ts:6]: Mainly for consistency as it looks off for some lines to be start with `-` and others not

**Comment by ijjk** [comment]
## Failing test suites 
Commit: c966abe97f3a4afacd35a772192ea120d3aa51c6

`pnpm testheadless test/e2e/edge-configurable-runtime/index.test.ts` 
- Configurable runtime for pages and API routes > In dev mode > errors about page using edge runtime
- Configurable runtime for src/pages and API routes > In dev mode > errors about page using edge runtime

<details>
<summary>Expand output</summary>

  ● Configurable runtime for src/pages and API routes › In dev mode › errors about page using edge runtime

    expect(received).toInclude(expected)

    Expected string to include:
      "error - Page /pages provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead."
    Received:
      "yarn run v1.22.19
    $ /tmp/next-install-ce26ebfb8dc74518ef2dbbe360836268ba193c3c0847e7df7e6ea8268bb66756/node_modules/.bin/next
    - ready started server on 0.0.0.0:33123, url: http://localhost:33123
    - info Using locally built binary of @next/swc
    - event compiled client and server successfully in 1813 ms (151 modules)
    - error Page /pages provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - error Page / provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - wait compiling / (client and server)...
    - error Page / provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - error Page / provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - event compiled client and server successfully in 1044 ms (320 modules)
    - error Page / provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - info Using locally built binary of @next/swc
    - error Page /pages provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - error Page / provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - GET / 200 in 862ms
    "

      86 |         const res = await fetchViaHTTP(next.url, `/`)
      87 |         expect(res.status).toEqual(200)
    > 88 |         expect(stripAnsi(next.cliOutput)).toInclude(
         |                                           ^
      89 |           `error - Page /pages provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.`
      90 |         )
      91 |         expect(next.cliOutput).not.toInclude('warn')

      at Object.<anonymous> (e2e/edge-configurable-runtime/index.test.ts:88:43)

  ● Configurable runtime for pages and API routes › In dev mode › errors about page using edge runtime

    expect(received).toInclude(expected)

    Expected string to include:
      "error - Page /pages provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead."
    Received:
      "yarn run v1.22.19
    $ /tmp/next-install-da5bf242dff999b0169061b94ea31e3c74e5c4e24c5a69437a95990c283eeedf/node_modules/.bin/next
    - ready started server on 0.0.0.0:40549, url: http://localhost:40549
    - info Using locally built binary of @next/swc
    - event compiled client and server successfully in 1400 ms (151 modules)
    - error Page /pages provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - error Page / provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - wait compiling / (client and server)...
    - error Page / provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - error Page / provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - event compiled client and server successfully in 2.8s (320 modules)
    - error Page / provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - info Using locally built binary of @next/swc
    - error Page /pages provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    - error Page / provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.
    "

      86 |         const res = await fetchViaHTTP(next.url, `/`)
      87 |         expect(res.status).toEqual(200)
    > 88 |         expect(stripAnsi(next.cliOutput)).toInclude(
         |                                           ^
      89 |           `error - Page /pages provided runtime 'edge', the edge runtime for rendering is currently experimental. Use runtime 'experimental-edge' instead.`
      90 |         )
      91 |         expect(next.cliOutput).not.toInclude('warn')

      at Object.<anonymous> (e2e/edge-configurable-runtime/index.test.ts:88:43)

</details>


Read more about building and testing Next.js in [contributing.md](https://github.com/vercel/next.js/blob/canary/contributing.md).

**Review by huozhi — DISMISSED** [review]

**Review by huozhi — APPROVED** [review]

**Pull request merged**